### PR TITLE
Add EditProfile screen and styles

### PR DESCRIPTION
**Pull request by NCMoseley**
Boilerplate code

### CODE REVIEW

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Good work, Nate!!

Please remove all the commented out code. If you are going to use it later, then save it somewhere else.

Thank you!

**Comment by NCMoseley** [comment]
Comments removed! (except this one)

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
/* global require */ // This is to avoid a linting error with 'require'

remove that comment from js/screens/EditProfile/EditProfileContainer.js file.

Otherwise, it looks good. Thank you!

**Comment by NCMoseley** [comment]
Comment deleted.

File: js/screens/EditProfile/EditProfile.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+          style={styles.largeInput}
+          placeholder="0/250"
+        />
+      </View>
+    </View>
+    <Button color={colors.lightBlue} text={"Save"} func={func} />
+  </ScrollView>
+);
+EditProfile.propTypes = {
+  func: PropTypes.string.isRequired
```
Comment (siddharthparmar7) [inline js/screens/EditProfile/EditProfile.js]: I believe func is a function, not a string. Talk to me if you have any questions.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Thank you, Nate! 

Once the requested change is made, I will merge this branch.

**Comment by NCMoseley** [comment]
OK all good, thanks

**Review by siddharthparmar7 — APPROVED** [review]
Thank you!

**Pull request merged**